shards/watcher.go

```diff
@@ -135,12 +135,14 @@ func (s *DirectoryWatcher) scan() error {
 		ts[fn] = fi.ModTime()
 
 		fiMeta, err := os.Lstat(fn + ".meta")
-		if err != nil {
-			continue
-		}
-		if fiMeta.ModTime().After(fi.ModTime()) {
+		if err == nil && fiMeta.ModTime().After(ts[fn]) {
```

Comment:
can we just loop over the modtimes of `zoekt.IndexFilePaths` instead?

Reply:
good idea!